test/parallel/test-worker.js

```diff
@@ -8,6 +8,9 @@ if (isMainThread) {
   w.on('message', common.mustCall((message) => {
     assert.strictEqual(message, 'Hello, world!');
   }));
+  w.on('error', (err) => {
+    assert.fail(`worker failed unexpectedly: ${err}`);
```

Comment:
```suggestion
    assert.ifError(err);
```

Reply:
as the error callback itself is unexpected, why do we want to check again, rather than just fail the test?